lib/resources/r4/observation.yaml

```diff
@@ -554,19 +554,23 @@ fields:
     {
       "valueString": '2'
     }
+  example4: |
+    {
+      "valueDateTime": '2020-07-04T07:15:00Z'
+    }
   note: |
     <ul>
       <li> Vital Signs support writing observations with `valueQuantity` </li>
     </ul>
 
     <ul>
-      <li> Laboratory support `valueQuantity`, `valueCodeableConcept` and `valueString` </li>
+      <li> Laboratory support `valueQuantity`, `valueCodeableConcept`, `valueString` and `valueDateTime` </li>
```

Comment:
are we missing period(.) at the end?

Reply:
updated : https://github.com/cerner/fhir.cerner.com/commit/ff3fe4a9864edf890ffe8d1182919fee145fcd42#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR567